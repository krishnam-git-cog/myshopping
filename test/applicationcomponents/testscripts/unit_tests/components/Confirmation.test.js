import React from 'react';
import { createShallow } from '@material-ui/core/test-utils';
import Confirmation from '../../../../../src/components/confirmation/Confirmation';


describe('Confirmation test', () => {
    it("Component - Confirmation should match snapshot",  () => {
      let shallow = createShallow();
      const component = shallow(<Confirmation />);
      expect(component).toMatchSnapshot();
    });

});