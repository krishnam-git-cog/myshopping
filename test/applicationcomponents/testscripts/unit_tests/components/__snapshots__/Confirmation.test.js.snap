// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Confirmation test Component - Confirmation should match snapshot 1`] = `ShallowWrapper {}`;
